packages/core/upload/admin/src/components/AssetDialog/BrowseStep/index.js

```diff
@@ -80,6 +80,13 @@ export const BrowseStep = ({
   const assetCount = assets.length;
   const folderCount = folders.length;
 
+  const handleClickFolderCard = id => {
+    // Search query will always fetch the same results
+    // we remove it here to allow navigating in a folder and see the result of this navigation
+    onChangeSearch('');
+    onChangeFolder(id);
```

Comment:
And then spead the args again here: 
```suggestion
    onChangeFolder(...args);
```